src/styles/components/_sign-in-form.scss

```
@@ -0,0 +1,48 @@
+@import '../abstracts/_abstracts-dir';
+
+.sign-in-form {
+  display: flex;
+  flex-direction: column;
+  background-color: $silver;
+
+  &__header {
+    display: flex;
+    justify-content: center;
+    background-color: greenyellow;
+    margin-bottom: rem(24px);
+    padding: rem(24px) 0;
+  }
+
+  &__body {
+    margin-bottom: rem(24px);
+  }
+
+  &__footer {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    margin-bottom: rem(24px);
+  }
+
+  &__footer-item {
+    &:not(:last-child) {
+      margin-bottom: rem(8px);
+    }
+  }
+
+  &__footer-item > a,
+  &__footer-item > p {
```

We can select all one level child by this
```suggestion
  &__footer-item > * {
```